gbpservice/neutron/tests/unit/nfp/base_configurator/api/v1/controllers/test_controller.py

```diff
@@ -17,6 +17,7 @@
 import pecan
 from pecan import rest
 import webtest
+import zlib
 
 from gbpservice.nfp.base_configurator.api import root_controller
 
```

Comment:
Lets pull changes from https://github.com/oneconvergence/group-based-policy/pull/308/files into this PR